crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/match.py

```diff
@@ -236,3 +236,15 @@ def foo():
         "b",
     ]:
         pass
+
+match foo:
+    case [1, 2, *rest]:
+        pass
+    case [1, 2, *_]:
+        pass
+    case [
+        1,
+        2,
+        *rest,
+    ]:
+        pass
```

Comment:
Could you add test case for the following?
```python
    case [1, 2, * # comment
        rest]:
        pass
    case [1, 2, * # comment
        _]:
        pass
```

Reply:
Added!